recipe/meta.yaml

```
@@ -10,17 +10,17 @@ source:
   sha256: 5d20a3b9aa851ca56dd09c57bb6d790cbe331e3e4d4c86e8a16575c9400071b6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script:  {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   number: 0
+  skip: True  # [py<38]
```

```suggestion
  skip: True  # [py<37]
```
https://github.com/Kaggle/kagglehub/blob/v0.2.7/pyproject.toml#L22

---

Thanks for the suggestion! Is there a reason we correct to 3.7 rather than 3.8 considering we will only build for 3.8 and above either way? Sorry, some clarification would be helpful as I was under the understanding and impression that we only support 3.8 and up!